engravings.lua

```diff
@@ -0,0 +1,35 @@
+--DFHack:fix/engravings.lua
+--4/23/24
+--@lord_cereal

```

Comment:
you can remove these -- they are recorded in the git commit history and don't need to be duplicated here